modules/oe_multilingual_selection_page/oe_multilingual_selection_page.module

```diff
@@ -6,22 +6,3 @@
  */
 
 declare(strict_types = 1);
-
-/**
- * Implements hook_page_preprocess().
- */
-function oe_multilingual_selection_page_preprocess_language_selection_page_content(array &$variables): void {
```

Comment:
Why did you remove this? What was this used for? And how would it break a site that uses oe_multilingual and now doesn't have this code here?

Reply:
This preprocess was used for ecl-language list back in the days. It does not serve anything to the contrib module: language_selection_page I checked. This is a left over after a refactor we did in oe_theme where we changed the ecl-language-list since then.

Reply:
It's very difficult to predict what site is relying on this. I get that we dont use it anymore, and 99% others dont either but if somebody does rely on this preprocessing, they are gonna have a bad surprise. Put a comment above saying this is deprecated and can be removed in 3.x or something.